code/GLG_Instance.m

```diff
@@ -85,7 +85,7 @@ function GLG_Instance(Data,varargin)
 else
     save(fullfile(params.outdir,[filename]), 'Adj_Matrix','varargin','params');
 end
-fprintf('file saved')
+fprintf('intermediate file saved')
```

Comment:
We may want a newline here.  Otherwise I get:
```
intermediate file savedWarning: Directory already exists. 
> In GLG_Instance (line 43)
  In SCINGE (line 22)
  In SCINGE_Example (line 24)
```

Reply:
Done. Will be part of next push.